src/contracts/index.html

```diff
@@ -45,14 +45,15 @@
       To do this, most require data access. These data plans are available to agencies with the pre-negotiated rates listed.
 ---
 
-<p>
-  The State of California has Master Service Agreements (MSAs) available to help you modernize your transit system.
-  We recommend creating an action plan for becoming GTFS-ready before going contactless, as the vendors you select
-  can help you use this data for reporting purposes plus help your customers see your real-time vehicle location and
-  fare information on journey-planning smartphone maps and apps.
+<p class="pb-3">
+  The State of California has Master Service Agreements (MSAs) available to help you modernize your transit system.  To add contactless fare collection, you will need up to three new technologies and ensure internet connectivity on your fleet (if you don’t already have it).
 </p>
 
-<p class="my-4">
+<strong class="h4">Become GTFS Compliant first for best results</strong> <em class="italics">Highly recommended</em>
```

Comment:
Just wanted to note here @mrose914 @esquared415 that `GTFS Compliant` and `GTFS-compliant` are formatted in two different ways on this copy as it is now:

<img width="882" alt="image" src="https://user-images.githubusercontent.com/3673236/206062832-68d28d7c-90ac-4e79-87be-a5cbe3097c5c.png">